website/blog/2022-10-05-aluminium.md

```diff
@@ -0,0 +1,805 @@
+`Metals: Start Scala CLI BSP server` and stopped with
+`Metals: Stop Scala CLI BSP server`
+
+If the script is in a dedicated folder, by default we will treat all the scripts
+and scala files in that directory as ones that can be used together. So you
+would be able to import method and classes from those files. However, if the
+script is contained within a directory that also contains other sources, that
+script will be treated as a standalone one in order to avoid flaky compilation
+errors coming from normal files in the workspace.
+
+## [Scala 3] Auto import symbols in string interpolation
+
+Previously, the only suggestions for string interpolations were coming from the
+currently available symbols in scope. This meant that if you wanted to import
+something from another package, you would need to do it manually.
+
+This problem is now resolved. Users can easily get such symbols automatically
+imported, which creates a seamless workflow.
+
+![auto-import](https://i.imgur.com/COKtciq.gif)
+
+These completions will also allow you to automatically import extension methods.
+
+The feature is already available for Scala 2.
+
+## [Scala 3] Add autofill named arguments completion
+
+Thanks to great work by [jkciesluk](https://github.com/jkciesluk) Scala 3
+completions will also show `autofill all` completion item, which will try to
+fill in all the arguments of the current method using the symbol in scope or
+with `???` if no symbols match. Users can later tabulate over the different
+proposal to adjust them.
+
+![auto-fill](https://i.imgur.com/tZTKnSw.gif)
+
+The feature was already available for Scala 2.
+
+## [Scala 3] Improved performance
+
+We've noticed that previous versions of Metals could cause CPU spikes for anyone
+using Scala 3. To address this, we managed to include a couple performance
+improvements in this release. This will improve several editor features such as
+hover, completions, document highlight, and some code actions.
+
+For more details, see the pull requests.
+
+- [Don't run InteractiveDriver.run if the content didn't "change" by tanishiking · Pull Request #4225 · scalameta/metals](https://github.com/scalameta/metals/pull/4225)
+- [fix: \[Scala3\] Don't compute docs on textDocument/completion by tanishiking · Pull Request #4396 · scalameta/metals](https://github.com/scalameta/metals/pull/4396)
+
+Thanks to [tanishiking](https://github.com/tanishiking)) for investigation and
+his great fixes.
+
+## Make documentHighlight more reliable
+
+Document highlight allows users to see all occurrences of the symbol currently
+under the cursor in a single file. This was one of the first
+[features](https://scalameta.org/metals/blog/2019/04/12/mercury#document-highlight)
+developed by me for Metals and previously it would only work in a fully compiled
+workspace. This means that if users have errors, some of the occurrences might
+not be highlighted.
+
+Since the feature only works within a single file, this was a perfect candidate
```

Comment:
I mentioned that we would require a fully compiling workspace above:
`developed by me for Metals and previously it would only work in a fully compiled
workspace.`

I had some info about semanticdb previously, but removed it since it might not be relevant for the users.